src/Aspire.Cli/Program.cs

```diff
@@ -214,6 +218,10 @@ public static async Task<int> Main(string[] args)
 
         using var app = await BuildApplicationAsync(args);
 
+        // Initialize the .NET runtime selector 
+        var runtimeSelector = app.Services.GetRequiredService<IDotNetRuntimeSelector>();
+        await runtimeSelector.InitializeAsync();
```

Comment:
We don't want to install the sdk at startup, only when the dotnet CLI is needed.